pkg/exporter/sumologicexporter/config.go

```diff
@@ -68,9 +68,6 @@ type Config struct {
 	// `procstat_num_threads` => `Proc_Threads` or `cpu_usage_irq` => `CPU_Irq`).
 	TranslateTelegrafMetrics bool `mapstructure:"translate_telegraf_attributes"`
 
-	// List of regexes for attributes which should be send as metadata
-	MetadataAttributes []string `mapstructure:"metadata_attributes"`
-
```

Comment:
hmm, we could make validation and exit program with meaningful error message for few releases